assets/js/components/ClusterDetails/ClusterInfoBox.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import ListView from '@components/ListView';
+import ProviderLabel from './ProviderLabel';
+
+const ClusterInfoBox = ({ haScenario, provider }) => (
+  <div className="tn-cluster-details w-full my-4 mr-4 bg-white shadow rounded-lg px-8 py-8">
+    <ListView
+      className="grid-flow-row"
+      titleClassName="text-lg"
+      orientation="vertical"
+      data={[
+        {
+          title: 'HA Scenario',
+          content: haScenario === 'hana_scale_up' ? 'HANA scale-up' : 'Unknown',
+        },
+        {
+          title: 'Provider',
+          content: provider || 'Not defined',
```

Comment:
Still not convinced with this `Not defined` string that is not used anywhere.
What happens if we remove it?
Or we put an empty string maybe otherwise...